contrib/meson.build

```diff
@@ -3,10 +3,17 @@ if get_option('qubes')
   subdir('qubes')
 endif
 
-if host_machine.system() == 'windows'
-  con2 = configuration_data()
-  con2.set('FWUPD_VERSION', fwupd_version)
+con2 = configuration_data()
+con2.set('FWUPD_VERSION', fwupd_version)
+con2.set('FWUPD_PLUGINVER', libfwupdplugin_lt_current)
+
+configure_file(
+  input : 'fwupd.spec.in',
+  output : 'fwupd.spec.in',
```

Comment:
shouldn't the output be `fwupd.spec?`

Reply:
Actually .in -- it's preprocessed again by a different tool... :)